src/gui/widgets/SimpleTextFloat.cpp

```diff
@@ -45,13 +45,44 @@ SimpleTextFloat::SimpleTextFloat() :
 
 	m_textLabel = new QLabel(this);
 	layout->addWidget(m_textLabel);
+
+	m_showTimer = new QTimer();
+	m_showTimer->setSingleShot(true);
+	QObject::connect(m_showTimer, &QTimer::timeout, this, &SimpleTextFloat::show, Qt::AutoConnection);
```

Comment:
Since ``Qt::AutoConnection``, we shouldn't have to specify it explicitly.
```suggestion
	QObject::connect(m_showTimer, &QTimer::timeout, this, &SimpleTextFloat::show);
```

Reply:
Here's another redundant use of ``Qt::AutoConnection`` you might've missed.